### PR TITLE
Create UK Wellcome Genome Campus hackathon page

### PR DESCRIPTION
**Pull request by priyanka-surana**

### CODE REVIEW

**Review by mribeirodantas — APPROVED** [review]

**Comment by mribeirodantas** [comment]
@nf-core-bot fix linting

File: src/content/events/2024/hackathon-march-2024/UK-WGC.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: Hackathon - March 2024 (Cambridge)
+subtitle: Local node of the nf-core hackathon at the Wellcome Genome Campus, Cambridge.
+type: hackathon
+startDate: '2024-03-18'
+startTime: '09:30+01:00'
```
Comment (mashehu) [inline src/content/events/2024/hackathon-march-2024/UK-WGC.md:6]:
```suggestion
startTime: '09:30+00:00'
```

File: src/content/events/2024/hackathon-march-2024/UK-WGC.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+endDate: '2024-03-19'
+endTime: '16:30+01:00'
```
Comment (mashehu) [inline src/content/events/2024/hackathon-march-2024/UK-WGC.md:8]:
```suggestion
endTime: '16:30+00:00'
```

**Review by mashehu — APPROVED** [review]
just some timezone changes needed, I guess.

**Comment by priyanka-surana** [comment]
How long before it shows up on the website? Thanks.

**Comment by mribeirodantas** [comment]
It's already there 😉

**Pull request merged**